sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/Generated/Models/CassandraClusterCommandPublicParameters.cs

```diff
@@ -0,0 +1,130 @@
+        /// <param name="serializedAdditionalRawData"> Keeps track of any properties unknown to the library. </param>
+        internal CassandraClusterCommandPublicParameters(string command, string commandId, BinaryData arguments, string host, bool? isAdmin, bool? cassandraStopStart, bool? readWrite, string result, CommandStatus? status, string outputFile, IDictionary<string, BinaryData> serializedAdditionalRawData)
+        {
+            Command = command;
+            CommandId = commandId;
+            Arguments = arguments;
+            Host = host;
+            IsAdmin = isAdmin;
+            CassandraStopStart = cassandraStopStart;
+            ReadWrite = readWrite;
+            Result = result;
+            Status = status;
+            OutputFile = outputFile;
+            _serializedAdditionalRawData = serializedAdditionalRawData;
+        }
+
+        /// <summary> The command which should be run. </summary>
+        public string Command { get; }
+        /// <summary> The unique id of command. </summary>
+        public string CommandId { get; }
+        /// <summary>
+        /// The arguments for the command to be run
+        /// <para>
+        /// To assign an object to this property use <see cref="BinaryData.FromObjectAsJson{T}(T, System.Text.Json.JsonSerializerOptions?)"/>.
+        /// </para>
+        /// <para>
+        /// To assign an already formatted json string to this property use <see cref="BinaryData.FromString(string)"/>.
+        /// </para>
+        /// <para>
+        /// Examples:
+        /// <list type="bullet">
+        /// <item>
+        /// <term>BinaryData.FromObjectAsJson("foo")</term>
+        /// <description>Creates a payload of "foo".</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromString("\"foo\"")</term>
+        /// <description>Creates a payload of "foo".</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromObjectAsJson(new { key = "value" })</term>
+        /// <description>Creates a payload of { "key": "value" }.</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromString("{\"key\": \"value\"}")</term>
+        /// <description>Creates a payload of { "key": "value" }.</description>
+        /// </item>
+        /// </list>
+        /// </para>
+        /// </summary>
+        public BinaryData Arguments { get; }
+        /// <summary> IP address of the cassandra host to run the command on. </summary>
+        public string Host { get; }
+        /// <summary> Whether command has admin privileges. </summary>
+        public bool? IsAdmin { get; }
+        /// <summary> If true, stops cassandra before executing the command and then start it again. </summary>
+        public bool? CassandraStopStart { get; }
+        /// <summary> If true, allows the command to *write* to the cassandra directory, otherwise read-only. </summary>
+        public bool? ReadWrite { get; }
```

Comment:
```suggestion
        public bool? IsReadWrite { get; }
```

Reply:
This is a property name used in our backend, maybe better to keep it.